BruTile.Common.props

```diff
@@ -22,4 +22,30 @@
     <NoWarn>1701;1702;1705;1591</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Label="Version numbers">

```

Comment:
I think this is the main purpose of your merge request. I created conflicts with my recent changes. I will merge just this part.